packages/rpc/src/RpcSerialization.ts

```diff
@@ -73,6 +75,152 @@ export const ndjson: Effect.Effect<RpcSerialization["Type"]> = Effect.sync(() =>
   })
 })
 
+/**
+ * @since 1.0.0
+ * @category serialization
+ */
+export const jsonrpc: Effect.Effect<RpcSerialization["Type"]> = Effect.sync(() => {
+  const decoder = new TextDecoder()
+  return RpcSerialization.of({
+    contentType: "application/json-rpc",
```

Comment:
Some JSON RPC server will NOT accept `application/json-rpc` (as ridiculous as that sounds).

In these cases, we could either require the user to override the header in their client or alternatively we'd have to change how the muxing currently behaves when e.g. `application/json` is set and instead make that more explicitly configurable here (I think I'd prefer that).